packages/manager/modules/hub/src/components/products/translations/Messages_de_DE.json

```diff
@@ -58,5 +58,6 @@
   "manager_hub_products_see_more": "Mehr Dienste anzeigen",
   "manager_hub_products_see_all": "Alles anzeigen",
   "manager_hub_products_OVH_CLOUD_CONNECT": "OVHcloud Connect",
-  "manager_hub_products_ESSENTIALS": "Managed Bare Metal"
+  "manager_hub_products_ESSENTIALS": "Managed Bare Metal",
+  "manager_hub_products_WEB_PAA_S_SUBSCRIPTION": "Web PaaS"
```

Comment:
Do we have any raison to write like this `PAA_S` ?

Reply:
Well this depends on the camel case and how the product is named. The product name is `Web PaaS` so i am getting `WEB_PAA_S`. Unfortunately, could not find a workaround for this.

Reply:
I understand your point of view.
I think we can let `_PAAS_` instead of `_PAA_S_` to preserve the meaning.

Waht do you think @varun257 ?

Reply:
Unfortunately, the key `manager_hub_products_WEB_PAA_S_SUBSCRIPTION` is generated from the product name which comes from /hub so i have no control over it :)